platforms/hyperledger-indy/configuration/roles/create/helm_component/ledger_txn/tasks/nested_main.yaml

```diff
@@ -19,8 +19,8 @@
             
 
     endorser_name={{ identity_name }}
-    endorser_did=$(curl --header "X-Vault-Token: {{ ac_vault_tokens[new_component_name] }}" "{{ new_org_vault_url }}"/v1/"{{ new_component_name }}"/endorser/"{{ identity_name }}"/identity/public | jq -r 'if .errors then . else .data.did end');  
-    endorser_verkey=$(curl --header "X-Vault-Token: {{ ac_vault_tokens[new_component_name] }}" "{{ new_org_vault_url }}"/v1/"{{ new_component_name }}"/endorser/"{{ identity_name }}"/node/public/verif_keys | jq -r 'if .errors then . else .data."verification-key" end');
+    endorser_did=$(curl --header "X-Vault-Token: {{ ac_vault_tokens[new_component_name] }}" "{{ new_org_vault_url }}"/v1/"{{ new_component_name }}"/endorsers/"{{ identity_name }}"/identity/public | jq -r 'if .errors then . else .data.did end');  
```

Comment:
I think we need also a identity_type, as we might want to use this chart to add an identity with a role other than ENDORSER